src/frequenz/dispatch/_dispatcher.py

```diff
@@ -13,22 +13,22 @@
 from frequenz.dispatch._event import DispatchEvent
 from frequenz.dispatch.actor import DispatchingActor
 
-ReceivedT = TypeVar("ReceivedT")
```

Comment:
The channels dependency changed to covariant types, so we have to follow suit

Reply:
Yeah, by now maybe we should move `ReceiverFetcher` to `channels`, it proven to be useful.